dec/decode_test.go

```diff
@@ -0,0 +1,73 @@
+package dec
+
+import (
+	"bytes"
+	"testing"
+
+	"gopkg.in/kothar/brotli-go.v0/enc"
+)
+
+func TestStreamDecompression(T *testing.T) {
+
+	input1 := bytes.Repeat([]byte("The quick brown fox jumps over the lazy dog. "), 100000)
+
+	output1 := make([]byte, len(input1)*2)
+	params := enc.NewBrotliParams()
+	params.SetQuality(4)
+
+	_, err := enc.CompressBuffer(params, input1, output1)
```

Comment:
It would be really cool to have a roundtrip test using both streaming interfaces - let's see about that when both interfaces are merged